gdcmodels/force_merge_segements.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import argparse
+import threading
+import urllib3
+
+from itertools import cycle
+from time import sleep, time
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionTimeout
+
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Force a merge on the shards of one or more ES indices"
+    )
+    parser.add_argument(
+        "--verify-certs",
+        default=False,
+        action="store_true",
+        help="Verify the ca certs for Elasticsearch",
+    )
+    parser.add_argument(
+        "--index",
+        help="The index to do the merge, if not provide, all indices will be merged.",
+    )
+    parser.add_argument(
+        "--host", required=True, help="Elasticsearch server host name or IP",
+    )
+    parser.add_argument(
+        "--port", default=9200, help="Elasticsearch server port (default: 9200)",
+    )
+    parser.add_argument("--user", default="", help="Elasticsearch client user")
+    parser.add_argument(
+        "--password", dest="password", default="", help="Elasticsearch client password"
+    )
+
+    return parser
+
+
+def _force_merge_wrapper(args):
+    es = Elasticsearch(
```

Comment:
you can probably reuse [this](https://github.com/NCI-GDC/gdc-models/blob/develop/gdcmodels/init_index.py#L80-L86)

Reply:
Without use_ssl=True, the request will fail for our es7. I guess previously we were using port 443 and this time we changed it to 9211?

Do you want me to modify the function in the init_index and add the flag?

Reply:
yes. you can make a generic enough method that'll return you an ES client and if you want to keep the old code working as it was, you can add a couple of params (including `use_ssl=True` and `port`) and have some defaults. but my guess would be that this needs to change anyway, because of recent xpack introduction